spring-cloud-stream-binder-kinesis/src/main/java/org/springframework/cloud/stream/binder/kinesis/config/KinesisBinderConfiguration.java

```diff
@@ -0,0 +1,58 @@
+package org.springframework.cloud.stream.binder.kinesis.config;
+
+import org.apache.commons.logging.Log;
+import org.apache.commons.logging.LogFactory;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.boot.autoconfigure.condition.ConditionalOnMissingBean;
+import org.springframework.boot.autoconfigure.context.PropertyPlaceholderAutoConfiguration;
+import org.springframework.boot.context.properties.EnableConfigurationProperties;
+import org.springframework.cloud.stream.binder.Binder;
+import org.springframework.cloud.stream.binder.kinesis.KinesisMessageChannelBinder;
+import org.springframework.cloud.stream.binder.kinesis.properties.KinesisBinderConfigurationProperties;
+import org.springframework.cloud.stream.binder.kinesis.properties.KinesisExtendedBindingProperties;
+import org.springframework.cloud.stream.binder.kinesis.provisioning.KinesisTopicProvisioner;
+import org.springframework.cloud.stream.config.codec.kryo.KryoCodecAutoConfiguration;
+import org.springframework.context.annotation.Bean;
+import org.springframework.context.annotation.Configuration;
+import org.springframework.context.annotation.Import;
+import org.springframework.integration.codec.Codec;
+
+/**
+ * 
+ * @author Peter Oates
+ *
+ */
+@Configuration
+@ConditionalOnMissingBean(Binder.class)
+@Import({KryoCodecAutoConfiguration.class, PropertyPlaceholderAutoConfiguration.class})
```

Comment:
Hi, no I don't see any issues when removing those imports. Although I also don't find any issues removing all 3 lines.

The last commit I pushed has them commented out, I can fix them once I know what should be set there.